test/functional/rpc_misc.py

```diff
@@ -55,7 +55,12 @@ def run_test(self):
 
         assert_raises_rpc_error(-8, "unknown mode foobar", node.getmemoryinfo, mode="foobar")
 
-        self.log.info("test logging")
+        self.log.info("test logging rpc and help")
+
+        # Test logging RPC returns the expected number of logging categories.
+        assert_equal(len(node.logging()), 35)
+
+        # Test toggling a logging category on/off/on with the logging RPC.
```

Comment:
22530: missing the actual sorting test https://github.com/bitcoin/bitcoin/pull/22530/files#diff-5da744cd4f92a7ee668493bfa03182254e8f375c4110ae5472212033d0f853d4R69-R77